ontology/yaml/resources/states/states.yaml

```diff
@@ -43,3 +44,13 @@ MATCHES: Two or more things match.
 SUBSCRIPTION_FAILED: The calendar events subscription has failed.
 UNKNOWN: The state is unknown.
 WAITING_FOR_RESPONSE: The calendar events synchronization process is awaiting response.
+
+# Added for measured statues (e.g. flowrate_status)
+PRESENT: "Something is currently materialized physically (e.g. flowrate_status is present)."
```

Comment:
being a little more verbose may add clarity: 
(e.g. in flowrate_status, flow is present)